library_generation/new_client/templates/owlbot.py.j2

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Google LLC
```

Comment:
This seems to be an existing bug. I see that we have modules generated [last year ](https://github.com/googleapis/google-cloud-java/blob/4982e78e416fec80baff99f0131815e2d4955acf/java-alloydb/owlbot.py#L1) with year `2021`. We need to at least update the copyright year to `2024` for all the templates for now, and come up with a way to always update it to current year.

Reply:
Sure, I'll update the year here.